lib/linters.py

```diff
@@ -0,0 +1,627 @@
+# Copyright 2008 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import namedtuple, defaultdict
+from copy import deepcopy
+import datetime
+import json
+import re
+
+from lib import nacaddr
+import yaml
+
+Error = namedtuple('Error', ['filename', 'lineno', 'offset', 'Severity',
+                             'message', 'category'])
+
+
+def sanitizeNetworkItem(x):
```

Comment:
This is an function name is incorrect. Lets try and stick with whats in the style guide. https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming